test/ruby/test_gc.rb

```diff
@@ -380,6 +380,11 @@ def test_gc_parameter
       assert_in_out_err([env, "-w", "-e", "exit"], "", [], /RUBY_GC_OLDMALLOC_LIMIT=6000000/, "")
       assert_in_out_err([env, "-w", "-e", "exit"], "", [], /RUBY_GC_OLDMALLOC_LIMIT_MAX=16000000/, "")
       assert_in_out_err([env, "-w", "-e", "exit"], "", [], /RUBY_GC_OLDMALLOC_LIMIT_GROWTH_FACTOR=2.0/, "")
+
```

Comment:
We should be able to write a test like this:

```ruby
  def test_aging_factor
    assert_separately([{ "RUBY_GC_AGING_FACTOR" => "1" }, "-W0", "--disable-gem", "-robjspace"], __FILE__, __LINE__, <<-RUBY)
      def run_test
        obj = Object.new

        4.times { GC.start(full_mark: false) }

        assert_include ObjectSpace.dump(obj), '"old":true'
      end

      run_test
    RUBY

    assert_separately([{ "RUBY_GC_AGING_FACTOR" => "4" }, "-W0", "--disable-gem", "-robjspace"], __FILE__, __LINE__, <<-RUBY)
      def run_test
        obj = Object.new

        4.times { GC.start(full_mark: false) }

        assert_not_include ObjectSpace.dump(obj), '"old":true'

        8.times { GC.start(full_mark: false) }

        assert_include ObjectSpace.dump(obj), '"old":true'
      end

      run_test
    RUBY
  end
```

Reply:
Added.